packages/core/src/components/select/test/select.ct.ts

```diff
@@ -172,3 +172,107 @@ test('open filtered dropdown on input', async ({ mount, page }) => {
   await expect(item1).toBeVisible();
   await expect(item2).not.toBeVisible();
 });
+
+test('remove text from input and reselect the element', async ({
+  mount,
+  page,
+}) => {
+  await mount(`
+        <ix-select value="2">
+          <ix-select-item value="1" label="Item 1">Test</ix-select-item>
+          <ix-select-item value="2" label="Item 2">Test</ix-select-item>
+          <ix-select-item value="3" label="Item 3">Test</ix-select-item>
+        </ix-select>
+    `);
+
+  await page.locator('[data-select-dropdown]').click();
+  const element = page.locator('ix-select');
+  await element.evaluate((select: HTMLIxSelectElement) => (select.value = []));
```

Comment:
Why do you need this line of code? If it serves no purpose, please remove it. I know I checked this part in my previous review but it seems I missed this one.

Reply:
we can not remove it, the purpose of it that  trying to update select value by programmatically,

Reply:
Can't the same thing be achieved by not setting `value="2"` in your html?

Reply:
I have to do it after rendering the component programatically

Reply:
What are you trying to test here? Is this to test whether the dropdown menu stays open after the value is removed programmatically?

Reply:
the main purpose is changing that updating dropdown value programmatically after first render, @jul-lam

Reply:
So the main purpose of the test is:

- render select with a selected value 2
- remove the value programmatically
- set value 2 again (by clicking on "Item 2")
- testing if value 2 is set by checking wether the input text is equal to "Item 2"

If this is the case, please remove 

```
await element.locator('input').fill('Item 3');

const item1 = page.getByRole('button', { name: 'Item 1' });
const item2 = page.getByRole('button', { name: 'Item 2' });
const item3 = page.getByRole('button', { name: 'Item 3' });

await expect(item1).not.toBeVisible();
await expect(item2).not.toBeVisible();
await expect(item3).toBeVisible();
```

since it is not part of the main test, and already testet in the "filter" test. Furthermore, the name of the test should be updated to represent the main test case.

Reply:
hi @jul-lam , you are right , I have just removed thanks :)

Reply:
Thank you! PR is approved from my side.